source/docs/articles/frequently-asked-questions.md

```diff
@@ -35,7 +37,7 @@ This is not officially supported, but the PHP runtime is complete. Some users ha
 
 Pantheon supports toggling between local development mode using `git push` to transfer all code changes, and an on-server development mode, which provides access to the codebase via SFTP.
 
-Direct SSH access is not supported, but you are able to directly interface with mysql, use CLI tools ([Terminus](https://github.com/pantheon-systems/cli), [drush](/docs/articles/local/drush-command-line-utility/), [WP-CLI](#does-pantheon-support-wp-cli?)), and SFTP files. Rsync is currently unstable on Pantheon. The engineering team is at work to improve its functionality on the platform.
+Direct SSH access is not supported, but you are able to directly interface with mysql, use CLI tools ([Terminus](https://github.com/pantheon-systems/cli), [drush](/docs/articles/local/drush-command-line-utility/), [WP-CLI](#Does-Pantheon-support-WP-CLI?), and SFTP files. Rsync is currently unstable on Pantheon. The engineering team is at work to improve its functionality on the platform.
```

Comment:
Rsync is a best practice, it is not unstable, it just creates temporary dotfiles faster than we can store and delete them, sometimes resulting in "ghost" files that aren't in Valhalla but are in fusedav. Workaround is to not store temp files in Valhalla during transmission, thus the required flag `--temp-dir=../tmp/`.